Capstone-Projects-2022-Spring/project-vicsr
# Vic 193 UI finalize design for document view

To test PR:

1. Login
2. See in document view that: The three parts of the dashboard align vertically; The document list shows all a user's associated documents; the document/flashcard nav bar at the bottom aligns with the bottom of the dashboard
